.github/workflows/wasmcloud.yml

```diff
@@ -493,18 +495,23 @@ jobs:
     strategy:
       matrix:
         crate:
+        - control-interface
         - provider-archive
         - wascap
         - wash-cli
         - wash-lib
 
-    if: startsWith(github.ref, 'refs/tags/') && github.event_name == 'push'
```

Comment:
nit: Now that we have the dry run, we could restrict the publish to only happen on the tag push right?

Does this action not being gated by tags mean we're just going to release new versions of the crate on merge? I don't think we've designed our workflow for that

Reply:
Crates are only published when the matching tag is pushed